crates/circuit/src/operations.rs

```diff
@@ -1406,6 +1875,21 @@ fn add_param(param: &Param, summand: f64, py: Python) -> Param {
     }
 }
 
+fn sum_params(param1: Param, param2: Param, py: Python) -> Param {
```

Comment:
Maybe `radd` instead of `sum`? I was a bit confused by this name at first because I knew we added `add` already for `PGate`.

Reply:
Absolutely, this was created before `add` and I didn't update the name. I renamed it to `radd_param` (singular for consistency) in [da3e598](https://github.com/Qiskit/qiskit/pull/12659/commits/da3e598b91c216cbddad3413990a90b3151196a9).

Reply:
The change has already been made but: Also, it seems customary in PLs to use "add" for two items, and "sum" for a list of items.